concordium-node/src/configuration.rs

```diff
@@ -511,6 +510,14 @@ pub struct ConnectionConfig {
         env = "CONCORDIUM_NODE_CONNECTION_DEDUPLICATION_HASHING_ALGORITHM"
     )]
     pub deduplication_hashing_algorithm: DeduplicationHashAlgorithm,
+    #[structopt(
+        long = "max-normal-keep-alive",
+        help = "Max seconds to hold connection to dead \"normal\" node before discarding. Must be \
+                at least 3 times greater than the value of \"housekeeping-interval\".",
```

Comment:
I guess this 3 is the same as `HOUSE_KEEPING_INTERVALS_PER_NORMAL_KEEP_ALIVE`. Can we use that here as well?

Reply:
Wasn't possible to format text with constant. Discarded part with the "3" instead after discussion with @abizjak .